math-scala/src/test/scala/org/apache/mahout/math/regression/OrdinaryLeastSquaresTest.scala

```diff
@@ -0,0 +1,64 @@
+package org.apache.mahout.math.regression
+
```

Comment:
We shouldn't need a new file- refactor the FTest to be like the coefficient of determintaiton / mse, move them to `FitnessTests` and add test to https://github.com/apache/mahout/blob/master/math-scala/src/test/scala/org/apache/mahout/math/algorithms/RegressionTestsSuiteBase.scala  with an R-Prototype